docs/conf.py

```diff
@@ -82,3 +82,7 @@
 }
 
 intersphinx_mapping = {"python": ("https://docs.python.org/3", None)}
+
+# Warn about all references to unknown targets
+nitpicky = True
```

Comment:
`nitpicky = True` is correct, but please remove it as we don't want to include until all links are fixed.